llvm/utils/update_test_checks.py

```diff
@@ -235,6 +235,7 @@ def main():
             )
         else:
             # "Normal" mode.
+            skip_same_check_line = False
```

Comment:
```suggestion
            dropped_previous_line = False
```
I think this name makes it easier to understand.